js/src/pages/product-feed/submission-success-guide/index.js

```diff
@@ -85,6 +98,77 @@ const pages = [
 				</p>
 			</GuidePageContent>
 		),
+		action: glaData.adsSetupComplete ? (
+			<AppButton
+				isPrimary
+				data-action="view-product-feed"
+				onClick={ handleGuideFinish }
+			>
+				{ __( 'View product feed', 'google-listings-and-ads' ) }
+			</AppButton>
+		) : undefined,
+	},
+	{
+		image,
+		content: (
+			<GuidePageContent
+				title={ __(
+					'Improve conversion tracking accuracy to improve campaign performance',
+					'google-listings-and-ads'
+				) }
+			>
+				<p>
+					{ __(
+						'Set up Enhanced Conversions, a feature designed to improve your measurement accuracy by collecting privacy-conscious data without the need for third-party cookies.',
+						'google-listings-and-ads'
+					) }
+				</p>
+				<p>
+					{ createInterpolateElement(
+						__(
+							'<link>Learn more</link> about Enhanced Conversions.',
+							'google-listings-and-ads'
+						),
+						{
+							link: (
+								<ContentLink
+									href="https://support.google.com/google-ads/answer/9888656"
+									context="enhanced-conversions"
+								/>
+							),
+						}
+					) }
+				</p>
+			</GuidePageContent>
+		),
+		actions: (
+			<AppButton
+				isPrimary
+				data-action="view-enhanced-conversions-settings"
+				eventName={ EVENT_NAME }
+				eventProps={ {
+					context: GUIDE_NAMES.SUBMISSION_SUCCESS,
+					action: 'view-enhanced-conversions-settings',
+				} }
+				onClick={ () => {
+					handleGuideFinish();
+					window.location.href = getSettingsUrl();
+				} }
+				aria-label={ __(
```

Comment:
@ankitrox Since the button label is clear and descriptive enough, let's remove the `aria-label` and `title` props for the button.